crates/ruff_python_parser/tests/snapshots/invalid_syntax@with_items_parenthesized_missing_colon.py.snap

```diff
@@ -0,0 +1,60 @@
+---
+source: crates/ruff_python_parser/tests/fixtures.rs
+input_file: crates/ruff_python_parser/resources/inline/err/with_items_parenthesized_missing_colon.py
+---
+## AST
+
+```
+Module(
+    ModModule {
+        range: 0..57,
+        body: [
+            With(
+                StmtWith {
+                    range: 28..56,
+                    is_async: false,
+                    items: [
+                        WithItem {
+                            range: 34..39,
+                            context_expr: Name(
+                                ExprName {
+                                    range: 34..39,
+                                    id: "item1",
+                                    ctx: Load,
+                                },
+                            ),
+                            optional_vars: None,
+                        },
+                        WithItem {
+                            range: 41..46,
+                            context_expr: Name(
+                                ExprName {
+                                    range: 41..46,
+                                    id: "item2",
+                                    ctx: Load,
+                                },
+                            ),
+                            optional_vars: None,
+                        },
+                    ],
+                    body: [
+                        Pass(
+                            StmtPass {
+                                range: 52..56,
+                            },
+                        ),
+                    ],
+                },
+            ),
+        ],
+    },
+)
+```
+## Errors
+
+  |
+1 | # `)` followed by a newline
+2 | with (item1, item2)
+  |                    ^ Syntax Error: Expected ':', found newline
+3 |     pass
+  |
```

Comment:
This is an improvement: https://play.ruff.rs/3c1fd581-e5df-4aa4-8528-7550e6d63e4f